card/spec/lib/card/reference_spec.rb

```diff
@@ -30,7 +30,7 @@
     end
 
     it "on template update" do
-      Card.create! :name=>"JoeForm", :type=>'UserForm'
+      Card.create! name: "JoeForm", type: 'UserForm'
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.